Exercise_C/index.html

```diff
@@ -10,7 +10,11 @@
   </head>
   <body>
     <!-- CODE HERE -->
-
+    <div class="container">
+      <div class="blue"></div>
+      <div class="green"></div>
+      <div class="red"></div>
+    </div>
     <script src="" async defer></script>
   </body>
 </html>
```

Comment:
Hey Chris, you might have added the wrong .css file. I think you can try attaching it once again.